src/popup/components/ListItemWrapper.vue

```diff
@@ -28,9 +28,7 @@ export default defineComponent({
 
 .list-item-wrapper {
   display: block;
-  padding: 8px var(--screen-padding-x);
-  margin-left: calc(-1 * var(--screen-padding-x));
-  margin-right: calc(-1 * var(--screen-padding-x));
```

Comment:
Why do you remove `calc`s here?

Reply:
this was adding -16px margins to left & right
and for padding the `var(--screen-padding-x)`is not working

Reply:
`var(--screen-padding-x)` is heavily used across our project, we shall adjust it everywhere.